packages/utils/src/dal/mikro-orm/mikro-orm-free-text-search-filter.ts

```diff
@@ -0,0 +1,163 @@
+import "@mikro-orm/core"
```

Comment:
suggestion: @adrien2p you could cleanup this code a bit to make it more readable please 🙏